internalsdk/session_model.go

```diff
@@ -2371,12 +2371,13 @@ func (session *SessionModel) appsAllowedAccess() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	strSlice := make([]string, len(installedApps))
-	for i, v := range installedApps {
-		strSlice[i] = fmt.Sprint(v.Value.PackageName)
+	var allowedAccess []string
+	for _, v := range installedApps {
+		if v.Value.AllowedAccess {
+			allowedAccess = append(allowedAccess, v.Value.PackageName)
+		}
 	}
-	return strings.Join(strSlice, ","), nil
-
+	return strings.Join(allowedAccess, ","), nil
```

Comment:
@jigar-f Looks like we were returning installed apps here instead of apps allowed access